### ISSUE TITLE
Headbutt data missing in version SoulSilver

### ISSUE THREAD

**Issue by leonardohra** [PokeAPI/pokedex#84]
So... I was playing SoulSilver and tried to export some data related to encounters. 
The problem is: I couldn't find any data on Headbutting trees on Pokemon Soul Silver.

Let's use Route 34 as an example.
You can see in serebii.net (https://www.serebii.net/pokearth/johto/route34.shtml) that you are able to find:

- Exeggcute 
- Hoothoot 
- Ledyba 
- Pineco

By headbutting trees, in version soulsilver.

If I try to check in PokeAPI what pokemons I can find in route 34 by headbutting trees (https://pokeapi.co -> location-area/johto-route-34-area -> [pokemon_encounters]), I can find info from a lot of games, except from SoulSilver.
Ok, maybe serebii.net isn't completely correct, and I shouldn't expect the API to return these 4 pokemons, but the API doesn't return any pokemon on soulsilver by headbutting trees.

Another way to check this is by viewing the raw json of the call mentioned before https://pokeapi.co -> location-area/johto-route-34-area and use "find" on "soulsilver", there will be some encounters there, but all encounters are listed as "walk" "old rod" "surf" and etc, no headbutts.

**Comment by Naramsim** [comment]
Yeah @leonardohra, probably we lack that kind of data :( You could check if veekun/pokedex has it
